<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallResponse</class>
 <widget class="QDialog" name="CallResponse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call request</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background: #ddd;
	color: #333;
}

QComboBox, QPushButton, QListWidget {
	border: 1px solid gray;
	border-radius: 5px;
}

QListWidget {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #F6F6F6, stop: 1.0 #EFEFEF);
}

QComboBox {
    padding: 1px 3px 1px 3px;
}

 QComboBox, QComboBox::drop-down:editable {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                  stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
 }

QComboBox:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

QComboBox:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}

 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
     border-width: 0px;
 }

QComboBox::down-arrow {
     image: url(/usr/share/icons/gnome/16x16/actions/down.png);
 }

QComboBox QAbstractItemView:editable::item:selected {
     alternate-background-color: gray;
 }

QPushButton {
	padding: 2px;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                  stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Call request from</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelContact">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAccept">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDecline">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Decline</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
